.ci/build.yml

```diff
@@ -0,0 +1,16 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: busybox
+
+inputs:
+  - name: rapidpro-docker
+
+outputs:
+  - name: image-params
+
+run:
+  path: ./rapidpro-docker/.ci/build.sh
+  params: {postfix: local}
```

Comment:
`params` needs to be a top level thing, and the build script gets the params as envvars. Alternatively, you could do something like this:
```
run:
  path: ./path/to/build.sh
  args:
    - {{version-suffix}}
```